apps/api/src/user-account/user.js

```diff
@@ -0,0 +1,66 @@
+const owasp = require('owasp-password-strength-test');
+const bcrypt = require('bcrypt');
+
+const config = require('../config');
+
+owasp.config({
+    allowPassphrases: true,
+    maxLength: 128,
+    minLength: 10,
+    minPhraseLength: 20,
+    minOptionalTestsToPass: 4,
+});
+
+/**
+ * Method to check if a string is a valid username
+ *
+ * @param {string} username
+ * @returns {object} an object with a boolean key isValid and an optionnal key error to describe error
+ */
+const isValidUsername = (username) => {
+    const usernameRegex = /^[a-zA-Z0-9-]{3,20}$/;
```

Comment:
```suggestion
    const usernameRegex = /^[\w-]{3,20}$/;
```

Reply:
Alors, je suis vraiment mauvais en regex :) Du coup, je ne peux t'avancer que cette argument : J'ai essayé cette modification, mais cela casse les tests ! Est-ce que cette regex prend bien en compte les chiffres ?

Reply:
Hum, en effet, la seule différence avec celle que tu as écris c'est que `\w` accepte les undercores (`\w Matches any letter, digit or underscore. Equivalent to [a-zA-Z0-9_].`. Après, vu que la regex est pas très compliqué, on peut la laisser comme ça...